Makefile

```diff
@@ -39,10 +39,12 @@ js:
 i18n:
 	$(PYTHON) ./scripts/i18n-messages compile
 
-git:
+git:	
+ifeq ($(DOCKER_HUB),FALSE)
```

Comment:
Let's add a comment explaining why we don't run these when using `DOCKER_HUB`

Reply:
Done.